### PR TITLE
202151035 Ayush Khandelwal

### PR DESCRIPTION
**Pull request by Ayush-Khandelwal28**
I have updated data.json

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   "data": [
     {
-      "name": "Rahul Bansal",
-      "batch": 2019,
-      "picture_url": "https://i.ibb.co/S0v6KWh/IMG-20201214-WA0000.jpg",
-      "twitter": "https://twitter.com/itz_rahul_rj",
-      "github": "https://github.com/rahulbansal0502",
-      "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
-      "quote": "Open source is 💓."
+      "name": "Ayush Khandelwal",
```
Comment (nlok5923) [inline data.json:93]: You are requested not to deleted the previous data instead add a new one.

**Review by nlok5923 — CHANGES_REQUESTED** [review]

**Comment by Ayush-Khandelwal28** [comment]
I have updated the file, and opened the pull request, can you please check
again and confirm if all done is correct or not?

Thanks
Ayush Khandelwal
202151035

File: data.json [hunk 2]
```diff
@@ -8,6 +8,15 @@
       "github": "https://github.com/rahulbansal0502",
       "linkedin": "https://www.linkedin.com/in/rahulbansal52/",
       "quote": "Open source is 💓."
+    } ,
```
Comment (nlok5923) [inline data.json]:
```suggestion
    },
```

**Review by nlok5923 — CHANGES_REQUESTED** [review]

**Comment by Ayush-Khandelwal28** [comment]
Updated
Please check now.
I am really sorry this is happening, it's my first time doing this.

Ayush Khandelwal
202151035

**Review by nlok5923 — APPROVED** [review]

**Pull request merged**